testing/test_browser.py

```diff
@@ -75,12 +78,6 @@ def test_wait_for_element_visible_fail_except(browser):
         browser.wait_for_element('#invisible_appear_p', visible=True, timeout=1.5)
 
 
-def test_wait_for_element_visible_fail_none(browser):
```

Comment:
since the option no longer exists, this test isn't relevant. Either the method returns a webelement or raises.